### PR TITLE
[explore] add "View samples" modal to action buttons

### PR DESCRIPTION
**Pull request by mistercrunch**
Also broke down the `View query` and `View results` as different
request so that viewing the query does not require fetching the results
anymore.
screen shot 2018-08-28 at 11 33 14 am
screen shot 2018-08-28 at 11 33 01 am

### CODE REVIEW

**Comment by mistercrunch** [comment]
@betodealmeida ping

File: superset/viz.py [hunk 1]
```diff
@@ -185,6 +185,17 @@ def get_fillna_for_columns(self, columns=None):
         }
         return fillna
 
+    def get_samples(self):
```
Comment (betodealmeida) [inline superset/viz.py]: Should we add a sort by random here? I guess it would be hard since it's database dependent...

**Review by betodealmeida — APPROVED** [review]
LGTM, just a comment on generating the samples.

File: superset/viz.py (hunk 1 above)
Comment (mistercrunch) [reply superset/viz.py]: Yup that would be database dependent and could be very expansive...

**Pull request merged**